src/parachain/issue.ts

```diff
@@ -149,21 +144,26 @@ export class DefaultIssueAPI implements IssueAPI {
 
     }
 
-    async execute(issueId: H256, txId: H256Le, merkleProof: Bytes, rawTx: Bytes): Promise<void> {
-        if (!this.account) {
-            return Promise.reject(ACCOUNT_NOT_SET_ERROR_MESSAGE);
-        }
-        const executeIssueTx = this.api.tx.issue.executeIssue(issueId, txId, merkleProof, rawTx);
-        await this.transaction.sendLogged(executeIssueTx, this.account, this.api.events.issue.ExecuteIssue);
+    async execute(requestId: string, btcTxId: string): Promise<void> {
```

Comment:
Let's make the `merkleProof` and the `rawTx` optional parameter. If someone really wants to fetch their own proos, we should allow that. There are some people that might not trust blockstream with their API and would prefer fetching this directly from BTC core.